src/controllers/notifications.js

```diff
@@ -86,16 +90,47 @@ exports.post = function * postNotification () {
   try {
     yield model.processNotification(notification, this.ledgers, this.config)
   } catch (e) {
-    if (!(e instanceof UnacceptableExpiryError)) {
+    if (
+      (e instanceof AssetsNotTradedError) ||
+      (e instanceof NoRelatedDestinationDebitError) ||
+      (e instanceof NoRelatedSourceCreditError) ||
+      (e instanceof UnacceptableConditionsError) ||
+      (e instanceof UnacceptableExpiryError) ||
+      (e instanceof UnacceptableRateError) ||
+      (e instanceof UnrelatedNotificationError)
+    ) {
+      // Certain exceptions indicate an error in the transfer memo, rather than
```

Comment:
// Certain exceptions indicate an error in the transfer memo, rather than
      // the transfer itself. Since the client is the ledger and the ledger is
      // not at fault here, it doesn't make sense to return a 4xx response. In
      // other words, both the ledger and connector have done everything right,
      // so the request status between them should be 200.

Status code 202 (The request has been accepted for processing, but the processing has not been completed) could also makes sense here -- it is a 'success' from the requester's perspective but also indicates some issue with the request.